### PR TITLE
Added Currency Converter 

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
### 🛠️ Fixes Issue (Number)
#309 

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots

Currency-converter

### CODE REVIEW

**Comment by NOiR-07** [comment]
Add your project in index file of the website too

**Comment by NOiR-07** [comment]
Update your forked repo too as there is some conflicts

File: index.html [hunk 1]
```diff
@@ -621,7 +621,18 @@ <h5 class="card-title">Simon Game</h5>
 </div>
 <!-- end of Simon Game -->
 
-
```
Comment (NOiR-07) [inline index.html:624]: undo this change

**Comment by NOiR-07** [comment]
can you make a new pr for the same?

**Comment by Sulagna-Dutta-Roy** [comment]
It's not working after changes. So I want to make a PR for the same.

**Comment by NOiR-07** [comment]
> It's not working after changes. So I want to make a PR for the same.

okk
